pages/whats-new/overview.md

```diff
@@ -12,12 +12,26 @@ columns:
   - title: Page views
     source: Pageviews
 redirect_from:
+  - /about/releases/
   - /about-our-work/
+  - /about-our-work/releases
   - /whats-new/
-in_page_nav: false
+  - /whats-new/releases
+subnav:
+- text: Latest updates
+  href: '#changelog'
+- changelog:
+  key: about-whats-new
 ---
 
-{% capture blog %}
+## Release notes
+
+You can find summaries of bug fixes, new features, and other updates introduced in each release on our [GitHub releases page](https://github.com/uswds/uswds/releases).
```

Comment:
New suggestion for first paragraph, but interested to know if @finekatie has any tweaks: 

Find summaries of each USWDS update on our [GitHub releases page](https://github.com/uswds/uswds/releases). Release notes explain bug fixes, new features, and other changes.

Reply:
Updated in [11a0ab7](https://github.com/uswds/uswds-site/pull/2318/commits/11a0ab7f4583c51362114a715cc49f201856b06b)